.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8.9
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          - v1-dependencies-
```

Comment:
It's for cache hit.
https://circleci.com/docs/2.0/caching/#source-caching

Reply:
Ah I see, thanks !

Now, I realize that we use `package.json`, we should use `yarn.lock` because upgrades with minor version change don't change `package.json` but just `yarn.lock`. Now that I understand how keys work, I think we can just add a line at the top using both `package.json` and `yarn.lock`:
```yaml
- restore_cache:
    keys:
    - v1-dependencies-{{ checksum "package.json" }}-{{ checksum "yarn.lock" }}
    - v1-dependencies-{{ checksum "package.json" }}
    - v1-dependencies-
```
(and use the first line for `save_cache`).